entropy/entropy.go

```diff
@@ -26,3 +30,46 @@ func GetRandom(source EntropySource, len uint32) ([]byte, error) {
 	}
 	return randomBytes, nil
 }
+
+// EntropyReader hold info for user entropy to be given to the dkg
+// Warning: UserOnly should be set to true for debugging, and randomness coming from user
+// should always be mixed with crypto/rand and thus userOnly dhoul be false by default
+type EntropyReader struct {
+	Entropy  string
+	UserOnly bool
```

Comment:
Why do you have the flag here in this struct ? It 's not used per se here. It's only dkg that uses it, so I would suggest to remove it from here. This struct has one job, to be a Reader from an executable.